pkg/webservice/state_dump.go

```diff
@@ -37,16 +37,16 @@ const (
 var stateDump sync.Mutex // ensures only one state dump can be handled at a time
 
 type AggregatedStateInfo struct {
-	Timestamp        int64
-	Partitions       []*dao.PartitionInfo
-	Applications     []*dao.ApplicationDAOInfo
-	AppHistory       []*dao.ApplicationHistoryDAOInfo
-	Nodes            []*dao.NodesDAOInfo
-	ClusterInfo      []*dao.ClusterDAOInfo
-	ContainerHistory []*dao.ContainerHistoryDAOInfo
-	Queues           []dao.PartitionQueueDAOInfo
-	RMDiagnostics    map[string]interface{}
-	LogLevel         string
+	Timestamp        int64                            `json:",omitempty"`
+	Partitions       []*dao.PartitionInfo             `json:",omitempty"`
+	Applications     []*dao.ApplicationDAOInfo        `json:",omitempty"`
+	AppHistory       []*dao.ApplicationHistoryDAOInfo `json:",omitempty"`
+	Nodes            []*dao.NodesDAOInfo              `json:",omitempty"`
+	ClusterInfo      []*dao.ClusterDAOInfo            `json:",omitempty"`
+	ContainerHistory []*dao.ContainerHistoryDAOInfo   `json:",omitempty"`
+	Queues           []dao.PartitionQueueDAOInfo      `json:",omitempty"`
+	RMDiagnostics    map[string]interface{}           `json:",omitempty"`
+	LogLevel         string                           `json:",omitempty"`
```

Comment:
we should set the json name for these fields inline with all our other objects that we create

Reply:
Hi @wilfred-s, thanks for the review. We didn't set json name, so the output key is like `Timestamp`, not `timestamp`. Do we want to add `timestamp` as json name, or `Timestamp`? Thank you.

Reply:
Updated it. Thank you.